libafl/src/fuzzer/mod.rs

```diff
@@ -387,7 +384,6 @@ where
 
                 // Add the input to the main corpus
                 let mut testcase = Testcase::with_executions(input.clone(), *state.executions());
-                testcase.set_parent_id_optional(state.fuzzed_corpus_id());
```

Comment:
Set the .current ID here

Reply:
i'll do it on_add

Reply:
Don't think that's correct, see my comment below (probably doesn't work for Objectives as they are a different corpus(?)

Reply:
@tokatoka ?

Reply:
yeah was busy last two days.
now I put this back